manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.installer.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://www.fjsoft.at/download.php?id=1

```

```suggestion
  InstallerUrl: https://www.fjsoft.atfiles/MyPhoneExplorer_Setup_1.8.15.exe
```
Please use static URLs if possible.